infor-design/enterprise
# 2428 - Improve a fix for re-measuring datagrid columns

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
When doing https://github.com/infor-design/enterprise/pull/2404 i fixed a problem but it introduced a behavior change thats unindented. This fixes.

**Related github/jira issue (required)**:
Fixes #2428  

**Steps necessary to review your pull request (required)**:
- retest #2404 
- But specifically go to http://localhost:4000/components/datagrid/test-editor-dropdown-source
- open the header multiselect and pick some items -> the columns should not change size when selecting
- go to http://localhost:4000/components/datagrid/test-paging-select-indeterminate-single.html
- note the width of the product name column
- go to last page -> the  product name column should adjust in size to the text

**Included in this Pull Request**:
